react-search-component/my-app/src/SearchableList Components/SearchableList.tsx

```diff
@@ -0,0 +1,24 @@
+import SearchBar from './SearchBar';
+import TextDisplay from './TextDisplay';
+import { useState } from 'react';
+
+type SearchableListProps = {
+  list: { id: string; sentence: string }[];
```

Comment:
You already defined List in App.tsx so you should import it and use it here. (You'll need to export the type in App.tsx.)

This is the DRY principle (Don't Repeat Yourself).

Reply:
Updated